mmv1/third_party/terraform/services/logging/resource_logging_project_bucket_config.go

```diff
@@ -282,6 +316,13 @@ func resourceLoggingProjectBucketConfigRead(d *schema.ResourceData, meta interfa
 		return fmt.Errorf("Error setting cmek_settings: %s", err)
 	}
 
+	indexConfigs, ok := res["indexConfigs"]
+	if ok {
```

Comment:
Same as above- we need to unconditionally write this so that we detect the removal of the value out-of-band.